spinn_front_end_common/utilities/system_control_logic.py

```diff
@@ -163,13 +163,12 @@ def _load_application(executable_targets, app_id):
     # Check that the binaries have reached a wait state
     count = transceiver.get_core_state_count(app_id, CPUState.READY)
     if count < executable_targets.total_processors:
-        cores_ready = transceiver.get_cpu_infos(
-            executable_targets.all_core_subsets, [CPUState.READY], False)
+        cores_ready = transceiver.get_cores_in_states(
+            executable_targets.all_core_subsets, [CPUState.READY])
```

Comment:
Interesting - in master this seems to hold "cores_ready" but actually it is "cores_not_ready"...  I think it is meant to show the status of things that are not ready though... (and then it should be able to use the status string)